CMakeLists.txt

```diff
@@ -304,7 +304,6 @@ endif()
 
 cxx_feature_check(STEADY_CLOCK)
 # Ensure we have pthreads
-set(THREADS_PREFER_PTHREAD_FLAG ON)
```

Comment:
Why is this being removed?

Reply:
Whether `-pthread` or `-lpthread` is the right choice is not something I am knowledgeable enough to weigh in on. It's been discussed here a few times (#919 #771) and at [googletest/#2482](https://github.com/google/googletest/issues/2482)), too. I wanted to make as few decisions as possible, so I removed this to allow the default behaviour (`@CMAKE_THREAD_LIBS_INIT@` will give `-lpthread`).

If these lines remained, I don't think there would be a way for someone to prefer `-lpthread` as `THREADS_PREFER_PTHREAD_FLAG ON` would always override it.

Reply:
I'm 99.99% sure this line should remain.